examples/complete/variables.tf

```diff
@@ -103,6 +103,24 @@ variable "arn_format" {
   default     = "aws"
 }
 
+variable "use_custom_dynamodb_table" {
+  type        = bool
+  description = "Boolean for when a user-created custom DynamoDB table is to be used"
+  default     = false
+}
+
+variable "dynamodb_table_name" {
+  type        = string
+  description = "Either the name of the user-created DynamoDB table, or a custom name for the default table"
+  default     = "geff-request-locking-table"
+}
+
+variable "custom_dynamodb_table_arn" {
```

Comment:
Are we make the the dynamodb an optional resource where it is created outside and input into the module or created within the module?